tech.json

```diff
@@ -2505,20 +2505,45 @@
+                "If Samus can overload PLMs, many types of blocks change their behavior: bomb, crumble, speed blocks, and sand behave as air (Power Bomb and shot blocks remain solid).",
+                "The most common ways to overload PLMs include: bombing Power Bomb and speed blocks and the side or bottom of crumble blocks that haven't been stepped on (approximately 40 times),",
+                "grappling grapple blocks, rolling through invisible camera scroll blocks (which often appear in morph tunnels), and touching sand.",
+                "Note that stepping on a crumble block or bombing a bomb block will prevent them from behaving like air after PLMs are overloaded.",
+                "When Samus enters G-mode, this is called direct G-mode if she leaves that room in any way, she is instead in indirect G-mode.",
+                "When in indirect G-mode, blue and green gates do not spawn, and Samus can freely walk through them."
               ],
               "extensionTechs": [
                 {
-                  "id": 200,
-                  "name": "canTrickyGMode",
+                  "name": "canComplexGMode",
                   "techRequires": [
                     "canEnterGMode"
                   ],
                   "otherRequires": [],
-                  "note": ["Doing something particularly precise or risky while in G-mode."],
-                  "devNote": "This could be expanded to unusual setups."
+                  "note": [
+                    "Doing something particularly precise, risky, or unintuitive while in G-mode or while setting up G-mode.",
+                    "This includes: enemy dodging that would not normally be an issue and is unintuitive, such as enemies that deal a small amount of damage and awkward dodging while artificially morphed;",
+                    "bypassing door locks while in G-mode to maintain indirect G-mode in the next room;",
+                    "exiting direct G-mode while in the open door to get very deep doorstuck and X-Ray climb and doorlock bypass a door above;",
+                    "exiting G-mode while inside of overloaded blocks in order to have them solidify and be able to X-Ray climb;",
+                    "touching an item in direct G-mode before PLMs are overloaded and obtaining the item remotely when exiting G-mode."
+                  ],
+                  "extensionTechs": [
+                    {
+                      "id": 200,
+                      "name": "canTrickyGMode",
+                      "techRequires": [
+                        "canComplexGMode"
+                      ],
+                      "otherRequires": [],
+                      "note": ["Doing something particularly precise, risky, or unintuitive while in G-mode or while setting up G-mode."]
```

Comment:
I think the `canTrickyGMode` description needs to be reworded to be different from `canComplexGMode`? Or maybe the `canComplexGMode` description should drop the word "particularly"?